.github/workflows/compile.yml

```diff
@@ -11,6 +11,7 @@ on:
 env:
   # Compiler defines common to all platforms
   COMMON_DEFINE: -DLLAMA_NATIVE=OFF -DLLAMA_BUILD_TESTS=OFF -DLLAMA_BUILD_EXAMPLES=OFF -DLLAMA_BUILD_SERVER=OFF -DBUILD_SHARED_LIBS=ON
+  LLAMA_CPP_COMMIT: a1d6df129bcd3d42cda38c09217d8d4ec4ea3bdd
```

Comment:
Good idea, I've thought about adding something like this but never quite got around to it.

Would it be possible to accept this as a `workflow_dispatch` input so it can be input from the UI instead of requiring a modification to this file?

Reply:
Split off into #470

Reply:
Merged :)

Reply:
Oh you'll probably need to rebase this onto master

Reply:
Yup, done.

Reply:
Probably want that github event variable fix first though #471

Reply:
My motiviation for #470 is that the current master in llama.cpp is broken, it causes the test host to segfault when loading a model.
I suspect they changed the model format again, making the current test model incompatible.